.pre-commit-config.yaml

```diff
@@ -28,15 +22,14 @@ repos:
     pass_filenames: false
     additional_dependencies:
       - Sphinx
-  - id: openapi-generator
-    name: OpenAPI-generator
-    language: node
-    entry: bash -c "cd .generator && openapi-generator-cli generate"
+  - id: generator
+    name: generator
+    language: python
+    language_version: "3.9"
+    entry: bash -c "cd .generator && python -m pip install . && python -m generator -i ./schemas/v1/openapi.yaml -o ../src/ && python -m generator -i schemas/v2/openapi.yaml -o ../src/"
```

Comment:
I would split it so we don't need to regenerate both v1 and v2 every time.

Reply:
Done.